vectordb_bench/backend/clients/pgvector/config.py

```diff
@@ -1,29 +1,64 @@
+from abc import abstractmethod
+from typing import Any, Mapping, Optional, Sequence, TypedDict
+
+from psycopg import sql
```

Comment:
Recommend avoiding adding specific dependencies in the `config.py`. Users **only install the corresponding toolkit when they are conducting tests.**

However, in the scenario where the default results page is opened solely for result display, `VDBBench` will load the standardized result (json). Serializing this data requires the `config.py` file from all clients.

Currently, if a user hasn't installed `psycopg`, they won't be able to open the results page.

Reply:
This is fixed in ea29f47